README.md

```diff
@@ -25,6 +25,7 @@ steps:
       packages: |
         package1.tar.gz
         package2.zip
+        packages/**/*.zip
```

Comment:
given that all these zip files are 0 byte files, it might be nicer if you just had the test script create them all on the fly with `fs.writeFileSync`, rather than having them all in git? 

It'd be easier to create different test combinations and wouldn't clutter up git?